prov/rxm/src/rxm_domain.c

```diff
@@ -204,7 +204,9 @@ fi_addr_t rxm_peer_av_ep_addr(struct fid_peer_av *av, struct fid_ep *ep)
 	if (ret)
 		goto err2;
 
-	return ofi_av_lookup_fi_addr(&rxm_av->util_av, addr);
+	ret = ofi_av_lookup_fi_addr(&rxm_av->util_av, addr);
```

Comment:
The return value is of type `fi_addr_t`, better to define a different variable for that instead of using `ret`.

Reply:
Done.